pygitguardian/incident_models/__init__.py

```diff
@@ -0,0 +1,138 @@
+from dataclasses import dataclass, field
+from datetime import datetime
+from typing import List, Optional, Type, Union, cast
+
+from marshmallow_dataclass import class_schema
+
+from pygitguardian.incident_models.constants import (
+    IncidentIgnoreReason,
+    IncidentSeverity,
+    IncidentStatus,
+    IncidentTag,
+    IncidentValidity,
+    OccurrenceKind,
+    OccurrencePresence,
+)
+from pygitguardian.models import Base, BaseSchema, FromDictMixin
+from pygitguardian.source_models import Source
+
+
+@dataclass
+class Detector(Base, FromDictMixin):
+    name: str
+    display_name: str
+    nature: str
+    family: str
+    detector_group_name: str
+    detector_group_display_name: str
+
+
+DetectorSchema = cast(Type[BaseSchema], class_schema(Detector, BaseSchema))
+Detector.SCHEMA = DetectorSchema()
+
+
+@dataclass
+class Match(Base, FromDictMixin):
```

Comment:
There is already a `Match` class in `models`, can we name this one `OccurrenceMatch`?